packages/block-library/src/video/editor.scss

```diff
@@ -71,36 +71,19 @@
 }
 
 .block-library-video-tracks-editor__track-list-track {
-	display: flex;
-	place-content: space-between;
-	align-items: baseline;
 	padding-left: $grid-unit-15;
 }
 
-.block-library-video-tracks-editor__single-track-editor-label-language {
-	display: flex;
-	margin-top: $grid-unit-15;
-	& > .components-base-control {
-		width: 50%;
-	}
-	& > .components-base-control:first-child {
-		margin-right: $grid-unit-20;
-	}
-}
-
 .block-library-video-tracks-editor__single-track-editor-kind-select {
 	max-width: 240px;
 }
 
 .block-library-video-tracks-editor__single-track-editor-buttons-container {
-	display: flex;
-	place-content: space-between;
-	margin-top: $grid-unit-40;
+	margin-top: $grid-unit-20;
```

Comment:
Curious about when we should use a Spacer component instead of a margin like in this instance, but it seems like it could be debatable. :)

Reply:
Far enough! The reasoning here was to get closer to matching trunk. If the narrow gap is okay, I'd rather remove the margin altogether. Or add Spacer unless that's also weird to have a gap and Spacer. But then I know you'd like to avoid Spacer in general! 😅

Reply:
Sorry I wasn't clear, we should maintain the same amount (32px) of whitespace. The point is to achieve it "logically", with a separate VStack, or a single margin-top:

<img width="605" alt="image" src="https://user-images.githubusercontent.com/555336/213193656-76ec45ac-b946-499b-8c34-ef8634f58616.png">

Reply:
Thanks for clarifying! I missed this comment earlier but my latest commit covers this now.